subsys/net/ip/net_context.c

```diff
@@ -1533,6 +1559,9 @@ int net_context_connect(struct net_context *context,
 	if (ret < 0) {
 		return ret;
 	}
+	/* TODO: Only in-case of ret = -ENOENT, should we wait for free
+	 * available connection handler until given 'timeout' ?
+	 */
```

Comment:
This comment is for another (API) matter, let's have it in a different patch addressing that issue.